RFC Template wording update (#2744)
DataDog/integrations-core: docs/proposals/TEMPLATE.md

## Problem

*If necessary, add a more detailed sketch of the problem. Add any notes,
pictures, text, or details that illuminate why this is a problem Datadog Agent 

> integrations community needs to solve. Keep it as concise as possible.*

Reviewer: `a problem that needs to be solved` maybe ?

## Constraints

*If necessary, note any constraints or requirements that any solution must
fulfill. For example:*

>     1. Datadog Agent has to target Python 2.7.

Reviewer: ```suggestion
    1. Target is Python 2.7.
```

    2. Memory footprint should not increase more than 10%.
    3. Every check in `integrations-core` and `integrations-extra` should be
       supported.

## Recommended Solution

*Describe your solution in the bare minimum detail to be understood. Explain

> why it's better than what Datadog Agent has and better than other options. Address

Author: not sure here about to who the `we` is referring to

Reviewer 2: "what we have" means "the current implementation" of whatever part of the repo

Author: changed

Reviewer: Is that better ?
```suggestion
why it's better than what's already in place and better than other options. Address
```
